compiler/test/runnable/powinline.d

```diff
@@ -0,0 +1,44 @@
+/*
+REQUIRED_ARGS: -O
+RUN_OUTPUT:
+---
+Success
+---
+*/
+import core.stdc.stdio;
+
+void test1()
+{
+    enum real Two = 2.0;
+    static assert(Two^^3 == 8.0);
+}
+
+void test2()
+{
+    double x = 5.0;
+    assert(x^^-1 == 1/x);
+    x = -1.0;
+    assert(x^^1 == x);
+    assert((x += 3) ^^ 2.0 == 4.0);
+    assert((x) ^^ 2.0 == 4.0);
+    assert((x *= 5) ^^ 2.0 == (x * x));
+    assert(x^^-1 == 1.0 / x);
+    assert((x^^-1) ^^ 0.0 == 1.0);
+}
+
+void test3()
+{
+    int x = 6;
+    assert(x ^^ 0 == 1);
+    assert((x += 3) ^^ 2 == 81);
+    assert(x ^^ 2 == (x ^^ 1) * (x ^^ 1));
+    static assert(4.0 ^^ -1 == 0.25);
+}
+
+void main()
```

Comment:
```suggestion
extern(C) void main()
```

In case my `-betterC` suggestion works.

Reply:
Addressed in the last commit, https://github.com/dlang/dmd/pull/21082/commits/72754cdebc284e7d87d8dd693d5b2d41a676fdf7.